layouts/partials/footer.html

```diff
@@ -22,5 +22,10 @@
         {{ end }}
       {{ end }}
     </section>
+    {{ if not .Site.Params.hidecolorschemetoggle }}
+    <span id="dark-mode-toggle" class="">
```

Comment:
I'm missing the on-hover effect here. Maybe changing from `span` to `a` should be enough. Also, please remove these empty `class` attribute.